app.controller.js

```diff
@@ -23,6 +23,16 @@ function getArticleId(req, res, next) {
 
 }
 
+function getArticles(req, res, next) {
+    getArticleData().then((articles) => {
```

Comment:
Consider renaming the model function to fall in line with convention using a synonym of get such as fetch or select.